File: Programmers/Lv0/문자열겹쳐쓰기/solution.py
def solution(my_string, overwrite_string, s):
    answer = ''
    end = s + len(overwrite_string) - 1

    for i in range(len(my_string)):
        if i < s or i > end:
            answer += my_string[i]
        else:
            answer += overwrite_string[i - s]

    return answer

File: Programmers/Lv0/문자열겹쳐쓰기/Solution.java
package Lv0.문자열겹쳐쓰기;

class Solution {
    public String solution(String myString, String overwriteString, int s) {
        StringBuilder sb = new StringBuilder();
        int end = overwriteString.length() + s - 1;

        for (int i = 0; i < myString.length(); i++) {
            if (i < s || i > end) {
                sb.append(myString.charAt(i));
            } else {
                sb.append(overwriteString.charAt(i - s));
            }
        }

        return sb.toString();
    }
}